03_ScienceExamples/MagellanicClouds/ExtremelyMetalPoorStarsInTheLMC.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    axs[0].set_ylabel('$m_i$')
</pre><p>m_i might be mislead as an apparent magnitude in i-band. Please improve clarity. For example, you can do "metallicity-sensitive index, m_i" or "m_i = (v-g)0 - 1.5(g-i)0". </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astro-datalab/notebooks-latest/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>